packages/notifications/src/styled/global-alert/StyledGlobalAlertButton.ts

```diff
@@ -6,65 +6,91 @@
  */
 
 import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
-import { getColorV8, DEFAULT_THEME, focusStyles } from '@zendeskgarden/react-theming';
+import {
+  getColor,
+  DEFAULT_THEME,
+  focusStyles,
+  retrieveComponentStyles
+} from '@zendeskgarden/react-theming';
 import { Button } from '@zendeskgarden/react-buttons';
 
 import { IGlobalAlertProps } from '../../types';
 import { colorStyles as basicColorStyles } from './StyledGlobalAlertClose';
 
+const COMPONENT_ID = 'notifications.global_alert.button';
+
 interface IStyledGlobalAlertButtonProps {
-  alertType: IGlobalAlertProps['type'];
+  $alertType: IGlobalAlertProps['type'];
   isBasic?: boolean;
 }
 
-function colorStyles(props: ThemeProps<DefaultTheme> & IStyledGlobalAlertButtonProps) {
-  if (props.isBasic) {
+type OffsetOptions = Record<string, Record<string, number>>;
+
+function colorStyles(
+  props: IStyledGlobalAlertButtonProps & ThemeProps<DefaultTheme> & IStyledGlobalAlertButtonProps
+) {
+  const { $alertType, isBasic, theme } = props;
+
+  if (isBasic) {
     return basicColorStyles(props);
   }
 
-  let backgroundColor;
-  let hoverBackgroundColor;
-  let activeBackgroundColor;
-  let focusColor;
+  let bgVariable;
+  let offsetOptions: OffsetOptions = { light: { offset: 200 }, dark: { offset: 300 } };
+  let offsetHoverOptions: OffsetOptions = { light: { offset: 300 }, dark: { offset: 400 } };
+  let offsetActiveOptions: OffsetOptions = { light: { offset: 400 }, dark: { offset: 500 } };
```

Comment:
Repeating from the `GlobalAlert` comment, I don't think we should be seeing light/dark offsets. The `GlobalAlert.Button` functions the same regardless of mode.

Reply:
Similar to my comment on the other comment, the offsets are there to ensure the underlying shade remains consistent between light/dark. If that isn't going to work here, we should definitely discuss in more detail.

Reply:
The direction I heard from @lucijanblagonic and @steue is that the components on the surface of a `GlobalAlert` do not change color (or transition hover/active state) differently in light & dark modes. I'm sure we want to use variables. But I wasn't expecting to see the `light` and `dark` parameters in use for `getColor` here.

Reply:
Oh, I see what you mean. The reason is that the values used in those variables between light and dark differs, so they need to be offset to meet the consistent shade in either mode. Hopefully I'm understanding correctly.